### DISCUSSION TITLE
Ground pins question

### DISCUSSION THREAD

**Discussion by andre2007** [onlaj/Piano-LED-Visualizer#507] [Q&A] [answered]
Hi, electrical newbie here 🙂

If I understand it correctly ground pin 6 and 9 of the raspberry are electrical connected. Would it work if I connect ground pin 6 with the minus pol of the "DC 5.5x2.5mm socket" and ground pin 9 with the ground cable of the led stripe?

**Comment by onlaj** [comment, marked as answer]
Yes, it should work just fine.

**Comment by andre2007** [comment]
Thanks a lot 😊
